cdk/PgStacInfra.ts

```diff
@@ -84,9 +72,10 @@ export class PgStacInfra extends Stack {
     // STAC API
     const stacApiLambda = new PgStacApiLambda(this, "pgstac-api", {
       apiEnv: {
```

Comment:
Just linking this for myself - https://stac-utils.github.io/stac-fastapi-pgstac/settings/#misc